client/src/modules/stock/entry/entry.html

```diff
@@ -73,6 +73,16 @@
               <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
             </div>
           </div>
+
+          <!-- funding source -->
+          <div ng-if="StockCtrl.stockSettings.enable_funding_source && StockCtrl.allowFundingSource">
```

Comment:
Could you make the `clear()` functionality work for this new feature?

![ClearNotWorking](https://github.com/user-attachments/assets/e773f501-0cba-4901-9599-6e79ce4bc733)

As a user, I expect that clicking the clear button should either reset the dropdown to a default value, or clear the value entirely.

Reply:
Similarly, when the value is `submit`-ed, it should clear this dropdown to prevent users from accidentally submitting the incorrect funding source.